{{cookiecutter.repo_name}}/{{cookiecutter.package_name}}/{{cookiecutter.package_name}}.py

```diff
@@ -69,3 +77,26 @@ def workbench_scenarios():
                 </vertical_demo>
              """),
         ]
+
+    @staticmethod
+    def _get_statici18n_js_url():
+        """
+        Returns the Javascript translation file for the currently selected language, if any.
+        Defaults to English if available.
+        """
+        lang_code = translation.get_language()
+        if not lang_code:
```

Comment:
Is that really possible, to have `lang_code` as falsy?

Reply:
Not False, but [None](https://docs.djangoproject.com/en/dev/ref/utils/#django.utils.translation.get_language) is possible. Better use `is None` instead.